fem/restriction.hpp

```diff
@@ -85,14 +95,19 @@ class ElementRestriction : public Operator
    /** Fill the J and Data arrays of SparseMatrix corresponding to the sparsity
        pattern given by this ElementRestriction, and the values of ea_data. */
    void FillJAndData(const Vector &ea_data, SparseMatrix &mat) const;
+   /// @private Not part of the public interface (device kernel limitation).
+   ///
+   /// Performs either MultTranpose or AddMultTranposes depending on the boolean
```

Comment:
typo: `MultTranspose`, `AddMultTranspose`

Reply:
Fixed in 6b38f5c46c96a097d28d374cf5e0fb2ab002e4d5